### PR TITLE
feat(statics): adds ofcsbc in coins.ts

### PR DESCRIPTION
**Pull request by shivagupta94**
Ticket: PX-2979

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -10124,6 +10124,7 @@ export const coins = CoinMap.fromCoins([
   ofcerc20('67fb4a6f-ba00-41fd-972d-728d2226a3d5', 'ofccho', 'Choise.com', 18, UnderlyingAsset.CHO),
   ofcerc20('af198c54-53d5-42b3-9e69-7fcc5887c0a0', 'ofcdia', 'DIA Token', 18, UnderlyingAsset.DIA),
   ofcerc20('4da7f6a6-9c56-44f7-a9b2-07ae7c669e42', 'ofcldo', 'Lido DAO Token', 18, UnderlyingAsset.LDO),
+  ofcerc20('4da7f6a6-9c56-44f7-a9b2-07ae7c669e42', 'ofcsbc', 'Sustainable Bitcoin Certificate', 8, UnderlyingAsset.SBC),
```
Comment (manav-c) [inline modules/statics/src/coins.ts:10127]: What does the first argument mean here? Seems like it should be different for every coin. Currently it's set to the same value as the coin above this one

Comment (Phani024) [inline modules/statics/src/coins.ts:10127]: The uuid is same as the uuid for `ofcldo` ^

Comment (shivagupta94) [reply modules/statics/src/coins.ts:10127]: Yeah, i missed that

**Review by Phani024 — CHANGES_REQUESTED** [review]
Blocking since the uuid is duplicate

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (shivagupta94) [reply modules/statics/src/coins.ts]: Changed

Comment (shivagupta94) [reply modules/statics/src/coins.ts]: Yeah, missed

**Review by Phani024 — APPROVED** [review]
LGTM

**Review by mandalarnav — APPROVED** [review]

**Pull request merged**